locales/en/app.ftl

```diff
@@ -514,6 +514,23 @@ want-to-add = Want to add another email?
 # the form to add an additional email to Firefox Monitor.
 verify-the-link = Verify the link sent to { $userEmail } to add it to {-product-name}.
 
+## These are part of a confirmation page that appears after a user has verified
+## an additional email to Firefox Monitor.
+
+email-verified = Email Successfully Verified!
+email-added-to-subscription = We’ll alert you if { $email } appears in a data breach.
+
+# This is the first part of the message that appears after the user has verified
+# their email address. It reads "To see and manage all emails you've signed up
+# for breach monitoring, sign in.
+email-verified-view-dashboard = To see and manage all emails you’ve signed up for breach monitoring, { $nestedSignInLink }.
+# This is the second part of the message that appears after the user has verified
+# their email address. It reads "To see and manage all emails you've signed up
+# for breach monitoring, sign in.
```

Comment:
```
# This message is used as a text for the subscribe link in email-verified-view-dashboard
```

Reply:
Updated.

Reply:
Would you like to have push/commit access this repo so we can make these kinds of suggestions/changes easier and faster?

And/or, for single-line suggestions, we could start using the "insert a suggestion" feature of GitHub PRs.

Reply:
I don't think I've ever used that feature, I need to figure out what it is first. As for commit access, not sure it would help a lot, because I wouldn't push changes to a branch where someone else is actively working.